x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/cache/shared/PartiallyCachedShardAllocationIntegTests.java

```diff
@@ -189,7 +189,6 @@ public void testOnlyPartialSearchableSnapshotAllocatedToDedicatedFrozenNodes() t
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/91800")
     public void testPartialSearchableSnapshotDelaysAllocationUntilNodeCacheStatesKnown() throws Exception {
```

Comment:
This test expect shard allocation is throttled while we collect available cache size across the nodes.
This is respected in computation (it produces an assignment with all shards ignored),
however this is silently using fallback shards during reconciliation.

This also confuses the followup computation as shards that ignored are unexpectedly started somewhere and the new balance is computed on top of this state.